src/ReduxStuffs/reducers/counselorReducer.js

```diff
@@ -0,0 +1,22 @@
+import initialState from '../initialState';
+import { SET_COUNSELOR } from '../actions';
+
+//action = {type: ACTION_TYPE, pyload: someDataHere}
+const counselorReducer = (state = {}, action) => {
+    if (action === undefined) return state;
```

Comment:
I think that maybe doing
```javascript
if (action === undefined) {
  console.warn('REDUCERS counselorReducer: action is undefined');
  return state;
}
```

Would be better to debug if at some point accidentally a dispatch is given without an action.
MAYBE